ticker/src/main/java/com/robinhood/ticker/TickerColumnManager.java

```diff
@@ -76,14 +94,17 @@ void setText(char[] text) {
         }
 
         // Use Levenshtein distance algorithm to figure out how to manipulate the columns
-        final int[] actions = LevenshteinUtils.computeColumnActions(getCurrentText(), text);
+        final int[] actions = LevenshteinUtils.computeColumnActions(
+                getCurrentText(), text, supportedCharacters
+        );
         int columnIndex = 0;
         int textIndex = 0;
-        for (int i = 0; i < actions.length; i++) {
-            switch (actions[i]) {
+        for (int action : actions) {
```

Comment:
change back to int-based for loop. the enhanced for loop allocates an iterator.

Reply:
Whoa good catch. The `DrawAllocation` lint didn't fire here since this isn't called in `onDraw()`.